app/services/articles/feeds/large_forem_experimental.rb

```diff
@@ -118,30 +102,26 @@ def more_experience_level_weight_experiment
       end
 
       def mix_of_everything_experiment
-        case rand(12)
+        case rand(10)
         when 0
           default_home_feed(user_signed_in: true)
         when 1
-          default_home_feed_with_more_randomness_experiment
-        when 2
-          mix_default_and_more_random_experiment
-        when 3
           more_tag_weight_experiment
-        when 4
+        when 2
           more_tag_weight_more_random_experiment
-        when 5
+        when 3
           more_comments_experiment
-        when 6
+        when 4
           more_experience_level_weight_experiment
-        when 7
+        when 5
           more_tag_weight_randomized_at_end_experiment
-        when 8
+        when 6
           more_experience_level_weight_randomized_at_end_experiment
-        when 9
+        when 7
           more_comments_randomized_at_end_experiment
-        when 10
+        when 8
           more_comments_medium_weight_randomized_at_end_experiment
-        when 11
+        when 9
           more_comments_minimal_weight_randomized_at_end_experiment
```

Comment:
Hm, just seeing this again makes me 😩 I wonder how we can improve this.

Reply:
You and me both which is why I decided to take ownership of it. We will get there!

Reply:
How about this?

Wrap `default_home_feed` in an `_experiment` method:

```ruby
def default_home_feed_experiment
  default_home_feed(user_signed_in: true)
end
```

Add this method to the `VARIANTS` constant and this becomes

```ruby
variant_method = VARIANTS.values.sample
public_send(variant_method)
```

If we don't want to add the wrapper, we can at least reduce it this to:

```ruby
if rand(10).zero?
  default_home_feed(user_signed_in: true)
else
  variant_method = VARIANTS.values.sample
  public_send(variant_method)
end
```

I prefer the first option with an explicit `default_home_feed_experiment`.

Reply:
@citizen428 my plan is to leave everything as is until we have "final paths" and then figure out the best way to handle those.